src/components/Credits/README.mdx

```diff
@@ -1 +1,4 @@
-## Usage
+## Help us improve!
```

Comment:
Can you please remove the help us improve, since we can add it via storybook for all components.